rclcpp/multithread_executor/package.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>examples_rclcpp_multithread_executor</name>
+  <version>0.0.0</version>
```

Comment:
This has to match the rest of the packages in this repository, so it should be `0.8.0.